README.md

```diff
@@ -1,35 +1,60 @@
 
-# Usage
-use `yarn test` to run tests
+*If you're having trouble understanding this contract, take a look at the [core Juicebox contracts](https://github.com/bananapus/juice-contracts-v4) and the [documentation](https://docs.juicebox.money/) first. If you have questions, reach out on [Discord](https://discord.com/invite/ErQYmth4dS).*
 
-use `yarn test:fork` to run tests in CI mode (including slower mainnet fork tests)
+## Develop
 
-<br>
+`juice-swap-terminal` uses the [Foundry](https://github.com/foundry-rs/foundry) development toolchain for builds, tests, and deployments. To get set up, install [Foundry](https://github.com/foundry-rs/foundry):
 
-use `yarn size` to check contract size
+```bash
+curl -L https://foundry.paradigm.xyz | sh
+```
 
-use `yarn doc` to generate natspec docs
+You can download and install dependencies with:
 
-use `yarn lint` to lint the code
+```bash
+forge install
+```
 
-use `yarn tree` to generate a Solidity dependency tree
+If you run into trouble with `forge install`, try using `git submodule update --init --recursive` to ensure that nested submodules have been properly initialized.
 
-<br>
+Some useful commands:
 
-use `yarn deploy:mainnet` and `yarn deploy:goerli` to deploy and verify (see .env.example for required env vars, using a ledger by default).
+| Command               | Description                                         |
+| --------------------- | --------------------------------------------------- |
+| `forge install`       | Install the dependencies.                           |
+| `forge build`         | Compile the contracts and write artifacts to `out`. |
+| `forge fmt`           | Lint.                                               |
+| `forge test`          | Run the tests.                                      |
+| `forge build --sizes` | Get contract sizes.                                 |
+| `forge coverage`      | Generate a test coverage report.                    |
+| `foundryup`           | Update foundry. Run this periodically.              |
+| `forge clean`         | Remove the build artifacts and cache directories.   |
 
-## Code coverage
-Run `yarn coverage`to display code coverage summary and generate an LCOV report
+To learn more, visit the [Foundry Book](https://book.getfoundry.sh/) docs.
 
-To display code coverage in VSCode:
-- You need to install the [coverage gutters extension (Ryan Luker)](https://marketplace.visualstudio.com/items?itemName=ryanluker.vscode-coverage-gutters) or any other extension handling LCOV reports
-- ctrl shift p > "Coverage Gutters: Display Coverage" (coverage are the colored markdown lines in the left gutter, after the line numbers)
+We recommend using [Juan Blanco's solidity extension](https://marketplace.visualstudio.com/items?itemName=JuanBlanco.solidity) for VSCode.
```

Comment:
Do we? Personnaly, I find hh ext overall better, including with foundry

Reply:
Great point. I'll update this in all the READMEs.